tests/test_meta/test_outlier_remover.py

```diff
@@ -6,7 +6,7 @@
 from sklearn.utils import estimator_checks
 
 from sklego.common import flatten
-from sklego.meta import OutlierRemover
```

Comment:
I think this test file should still point to `meta`. Otherwise we'll have two tests checking in the same code. The goal is to make sure that the original code in `meta` still works until we deprecate it.